practice-two/src/components/InputValidate/InputValidate.jsx

```diff
@@ -0,0 +1,54 @@
+import './InputValidate.css';
+import PropTypes from 'prop-types';
+import { memo } from 'react';
+
+const InputValidate = (props) => {
+  const {
+    label,
+    type = 'text',
+    value,
+    errorMessage,
+    genderType,
+    onChange,
+    onBlur,
+    ...inputProps
+  } = props;
+
+  return (
+    <div
+      className={`form__input ${
+        type !== 'radio' ? 'form__input--field' : 'form__input--choose'
+      }`}
+    >
+      {label && <label className='input__label'>{label}</label>}
+      <input
+        id={genderType}
+        type={type}
+        value={value}
+        className={type === 'radio' ? 'input__choose' : 'input__field'}
```

Comment:
create condition isRadioInput = type === 'radio' above and reuse

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/081c9cfa-23ea-49c3-b121-4b9b19ce351b)